### ISSUE TITLE
Can still add components to cameras post completion

### ISSUE THREAD

**Issue by MegaEmpirical** [tgstation/tgstation#41764]
Camera upgrades must now be added during the construction process https://github.com/tgstation/tgstation/pull/40533 but we're still able to add them post construction to no effect which is misleading for anyone who's learning/unfamiliar with the changes and serves no use, that I can think of.

**Comment by ShizCalev** [comment]
You should be able to upgrade them if the panel is screwed open

**Comment by MegaEmpirical** [comment]
Then what is the point of this PR: https://github.com/tgstation/tgstation/pull/40533

My understanding is that this means you must reconstruct a camera to upgrade them now and that's what I've had to do to get them to work now.

**Comment by ShizCalev** [comment]
@MegaEmpirical That PR was making it so that you can only upgrade them while they're attached to a wall, as opposed to upgrading them while they're in your hands.

Can you give steps for how you're trying to upgrade the camera?

**Comment by MegaEmpirical** [comment]
1) I place the camera on the wall
2) weld it
3a) wires 
3b) add any upgrades
4) screwdriver and select network

The PR said they can ONLY be upgraded during step 3 of construction, which seems to be the case whenever I've tried upgrading them. Any AI I've asked has said nothing has changed when I've tried feeding upgrades post construction. Proximity cameras don't light up or indicate anything if done after construction and so on.
